pkg/fleet/installer/packages/integrations/integrations_test.go

```diff
@@ -0,0 +1,131 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build !windows
+
+// Package integrations contains packaging logic for python integrations
+package integrations
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func TestRemoveCustomIntegrations(t *testing.T) {
+	dir := t.TempDir()
+
+	paths := []string{
+		// Installed by customer outside of the agent
+		"embedded/lib/python3.12/site-packages/botocore-1.38.8.dist-info/INSTALLER",
+		"embedded/lib/python3.12/site-packages/botocore-1.38.8.dist-info/METADATA",
+		"embedded/lib/python3.12/site-packages/botocore-1.38.8.dist-info/RECORD",
+		"embedded/lib/python3.12/site-packages/botocore-1.38.8.dist-info/REQUESTED",
+		"embedded/lib/python3.12/site-packages/botocore-1.38.8.dist-info/WHEEL",
+		"embedded/lib/python3.12/site-packages/botocore-1.38.8.dist-info/entry_points.txt",
+		"embedded/lib/python3.12/site-packages/botocore-1.38.8.dist-info/direct_url.json",
+		// Installed by agent integration command
+		"embedded/lib/python3.12/site-packages/datadog_activemq-5.0.0.dist-info/INSTALLER",
+		"embedded/lib/python3.12/site-packages/datadog_activemq-5.0.0.dist-info/METADATA",
+		"embedded/lib/python3.12/site-packages/datadog_activemq-5.0.0.dist-info/RECORD",
+		"embedded/lib/python3.12/site-packages/datadog_activemq-5.0.0.dist-info/REQUESTED",
+		"embedded/lib/python3.12/site-packages/datadog_activemq-5.0.0.dist-info/WHEEL",
+		"embedded/lib/python3.12/site-packages/datadog_activemq-5.0.0.dist-info/entry_points.txt",
+		"embedded/lib/python3.12/site-packages/datadog_activemq-5.0.0.dist-info/direct_url.json",
+		// Not installed by agent integration command
+		"embedded/lib/python3.12/site-packages/datadog_redisdb-5.0.0.dist-info/INSTALLER",
```

Comment:
similarly to the above, we expect redisdb to be removed so it was installed via the `datadog-agent integrations install` command and not present in the installed_by_pkg.txt